### PR TITLE
Update LodeRunner --help message default values

### PR DESCRIPTION
**Pull request by gortegaMS**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

Update LodeRunner --help message default values

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/703

### CODE REVIEW

File: docs/DataDictionary.md [hunk 1]
```diff
@@ -105,19 +105,19 @@ These are used for configuring a testing scenario.  `LoadTestConfig` will contai
 | EntityType      |   String   | Entity type used for filtering  | Yes | [`ClientStatus`, `LoadTestConfig`, `TestRun`] |
 | Id              |   String   | GUID used to retrieve the object directly. | Yes | |
 | Name            |   String   | Friendly name so that users may more easily identify configs | No | |
-| Files           |  String[]  | List of files to test   |   Yes     | match `--files` CLI flag |
+| Files           |  String[]  | List of files to test (default: `Null`)  |   Yes     | match `--files` CLI flag |
```
Comment (joheec) [inline docs/DataDictionary.md:108]: I think this is for LodeRunner in client mode, and the default shouldn't be null because a value is required

**Review by joheec — CHANGES_REQUESTED** [review]

File: docs/DataDictionary.md [hunk 2]
```diff
@@ -105,19 +105,19 @@ These are used for configuring a testing scenario.  `LoadTestConfig` will contai
...
-| RunLoop         |   Boolean  | Run test in an infinite loop (default: False) | No | match to `--run-loop` CLI flag |
-| Duration        |    Int     | Requires `RunLoop=True`. Test duration (seconds) (default: 0) | No | match to `--duration` CLI flag |
-| MaxErrors       |    Int     | Requires `RunLoop=False`. Max validation errors (default: 10) | No | match to `--max-errors` CLI flag |
-| DryRun          |   Boolean  | Validate the settings with the target clients (default `false`) | No | match to `--dry-run` CLI flag |
+| BaseURL         |   String   | Base url for files (default: `Null`) | No | match to `--base-url` CLI flag |
+| VerboseErrors   |   Boolean  | Displays validation error messages (default: `False`) | No | match to `--verbose-errors` CLI flag |
+| Randomize       |   Boolean  | Requires `RunLoop=True`. Dictates whether to process a load file top to bottom (default: `False`) or randomly | No | match to `--random` CLI flag |
+| Timeout         |    Int     | Request timeout in seconds (default: `30`) | No | match to `--timeout` CLI flag |
+| Server          |  String[]  | Server(s) to test (default: `Null`) | Yes | match to `--server` CLI flag |
```
Comment (joheec) [inline docs/DataDictionary.md:114]: I think this is for LodeRunner in client mode, and the default shouldn't be null because a value is required

**Review by joheec — CHANGES_REQUESTED** [review]

File: docs/DataDictionary.md (hunk 1 above)
Comment (gortegaMS) [reply docs/DataDictionary.md:108]: I removed (default: `Null`) for Files description.

File: docs/DataDictionary.md (hunk 2 above)
Comment (gortegaMS) [reply docs/DataDictionary.md:114]: I removed (default: `Null`) for Server description

**Pull request merged**